modules/test/portlet-title-localization-test/src/testIntegration/java/com/liferay/portlet/title/localization/test/PortletTitleTest.java

```diff
@@ -74,7 +74,7 @@ public void testPortletTitles() throws Exception {
 		Assert.assertTrue(
 			StringBundler.concat(
 				"Please update the Language.properties files for the ",
-				"following portlets: ", portletIdsWithMissingTitles.toString()),
+				"following portlets: ", portletIdsWithMissingTitles),
```

Comment:
Hey @MrJohn1911, it is not related to your changes. Please remove these changes from your pr. Thx

Reply:
@rafaprax Done!